chapter2/sxg/ex_2_5.pl

```diff
@@ -0,0 +1,7 @@
+#!perl
+print "Enter string: \n";
+$str = <STDIN>;
+print "Enter a number of times: ";
+chomp($num = <STDIN>);
+$result = $str x $num;
+print "The result is:\n$result";
```

Comment:
There are two 'newline's in the program. I thought they make it more readable.
Here is a sample run:
-------------------
Enter string: 
abc
Enter a number of times: 5
The result is:
abc
abc
abc
abc
abc